hamcrest-library/src/main/java/org/hamcrest/number/IsCloseTo.java

```diff
@@ -16,7 +16,7 @@
     private final double value;
 
     public IsCloseTo(double value, double error) {
-        this.delta = error;
+        this.delta = Math.abs(error);
```

Comment:
@nhojpatrick this should also be checked for null.

Reply:
@StarBax89 this file now lives at `./hamcrest/src/main/java/org/hamcrest/number/IsCloseTo.java`

if you rebase from the latest master, it should resolve the conflicts and I can merge in, e.g. `git fetch origin && git rebase origin/master`